charts/seed-monitoring/charts/grafana/values.yaml

```diff
@@ -18,9 +18,6 @@ vpaEnabled: false
 role: operators
 extensions:
   dashboards: ""
-  observedPods:
```

Comment:
In `docs/usage/logging.md` there is the following statement that is no longer valid after this change:

```
Exposing logs for a new component to the User's Grafana happens with adding a new `extensions.observedPods` section into: charts/seed-monitoring/charts/grafana/values.yaml
```

Can you adapt it?